cmd/lookoutd/serve.go

```diff
@@ -7,23 +7,25 @@ import (
 	"github.com/src-d/lookout/server"
 	"github.com/src-d/lookout/util/cli"
 	"github.com/src-d/lookout/util/ctxlog"
+
+	gocli "gopkg.in/src-d/go-cli.v0"
 )
 
 func init() {
-	if _, err := app.AddCommand("serve", "run a standalone server", "",
-		&ServeCommand{}); err != nil {
-		panic(err)
-	}
+	app.AddCommand(&ServeCommand{})
 }
 
 type ServeCommand struct {
+	gocli.PlainCommand `name:"serve" short-description:"run a standalone server" long-description:"Run a standalone server"`
 	queueConsumerCommand
 }
 
 func (c *ServeCommand) Execute(args []string) error {
```

Comment:
Let's use [ExecuteContext](https://github.com/src-d/go-cli/blob/master/context.go#L15) and remove our custom signal handling.

Reply:
Done in 5efef51